<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationConfigTab</class>
 <widget class="QWidget" name="LocationConfigTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="2" column="1">
    <widget class="QLineEdit" name="searchEdit">
     <property name="placeholderText">
      <string>Search . . .</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QLabel" name="mapLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>440</width>
       <height>220</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../icons/Icons.qrc">:/images/earthmap.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Current location</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_17">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="4" column="1">
       <widget class="ContextAwareSpinBox" name="altitudeSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>-500.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_52">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ContextAwareSpinBox" name="longitudeSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string>E </string>
        </property>
        <property name="suffix">
         <string> ยบ</string>
        </property>
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="countryCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_51">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="cityNameEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_53">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="ContextAwareSpinBox" name="latitudeSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string>N </string>
        </property>
        <property name="suffix">
         <string> ยบ</string>
        </property>
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_54">
        <property name="text">
         <string>Name / city</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_55">
        <property name="text">
         <string>Country</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QPushButton" name="addToListButton">
        <property name="text">
         <string>&amp;Add current location to list</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="cityListWidget"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>In order to perform accurate Doppler corrections of satellital signals, SigDigger needs to know the geographical coordinates of the reception site. Note that this does not apply to remote analyzers, as Doppler corrections are performed according to the remote receiver location.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
</widget>
<customwidgets>
   <customwidget>
     <class>ContextAwareSpinBox</class>
     <extends>QDouble</extends>
     <header>ContextAwareSpinBox.h</header>
   </customwidget>
  </customwidgets>

 <resources>
  <include location="../icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
